CHANGELOG.rst

```diff
@@ -1,3 +1,9 @@
+**2.3.0 - 12/19/23**
```

Comment:
updated CL here--will merge other two PRs and include them in the release

Reply:
Minor release seems right to me 👍